app/main.py

```diff
@@ -1 +1,52 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __str__(self) -> str:
+        return f"Name: {self.name}, " \
+               f"Health: {self.health}, " \
+               f"Hidden: {self.hidden}"
+
+    def __repr__(self) -> str:
+        return f"{{Name: {self.name}, " \
+               f"Health: {self.health}, " \
+               f"Hidden: {self.hidden}}}"
+
+    def is_hiding(self) -> bool:
+        return self.hidden
+
+    @staticmethod
+    def remove_dead_animals() -> None:
+        Animal.alive = [animal for animal in Animal.alive if animal.health > 0]
```

Comment:
There is no need for this `staticmethod`, because list comprehension with iteration would be excessive when you know what exact herbivore should be removed from the `alive` list in the `bite` method.